### PR TITLE
4.3.0 release

### PR DESCRIPTION
**Pull request by flotwig**
- 4.3.0 changelog
- New `sameSite` prop for cy.getCookie() and cy.getCookies() https://github.com/cypress-io/cypress-documentation/pull/2662
- New 'Experimental Features' section in Configuration. 
- [x] Error improvement docs https://github.com/cypress-io/cypress-documentation/pull/2683

### CODE REVIEW

File: source/_changelogs/4.3.0.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- We fixed a situation where a cross-origin errors could incorrectly throw in Chrome. Fixes {% issue 5270 %}.
+- Visibility checks against `vue-fragment` elements no longer throw an unexpected error. Fixes {% issue 6745 %}.
+- The stack trace is now properly shown in the console when clicking the command for tests having an `.only` that throw an error. Fixes {% issue 6744 %}.
+- `firefoxGcInterval` can now correctly be set to `null`. Fixes {% issue 6825 %}.
+- Cypress will no longer crash the process if an empty video chunk is received. Addressed in {% PR 6818 %}.
+
+**Misc:**
+
+- Security warnings no longer show in Mac OS when opening Cypress since our application now undergoes notarization from Apple. Addresses {% issue 5791 %}.
+- When an assertion involving a DOM element retry fails, it now shows a more sensible DOM object in the diff. Addresses {% issue 6723 %}.
```
Comment (brian-mann) [inline source/_changelogs/4.3.0.md]: Update the content and move this to `Bugfixes:`

- Assertions failures on DOM elements no longer show unexpected diffs when they weren't retried

File: source/_changelogs/4.3.0.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- When an assertion is retried (using {% url "`.should()`" should %}) and fails, it now shows the diff in the `stdout`. Addresses {% issue 6722 %}.
```
Comment (brian-mann) [inline source/_changelogs/4.3.0.md]: Move this to `Features:`

File: source/_changelogs/4.3.0.md (hunk 1 above)
Comment (flotwig) [reply source/_changelogs/4.3.0.md]: 69aa919a752d6e9bc82b33b10e8ace1c8d153b17

File: source/_changelogs/4.3.0.md (hunk 2 above)
Comment (flotwig) [reply source/_changelogs/4.3.0.md]: 69aa919a752d6e9bc82b33b10e8ace1c8d153b17

**Pull request merged**